packages/layout/src/footnotes/mapFootNotesToView.js

```diff
@@ -0,0 +1,59 @@
+
+import createInstance from '../node/createInstance';
+
+/**
+ *  maps footnote elements to a view node 
+ * 
+ * @param {*} footnotes 
+ * @returns a view node
+ */
+function mapFootNotesToView(footnotes) {
+  let processed = [
```

Comment:
Can you use `const` at all places where a certain variable is not supposed to get reassigned?